administrator/language/en-GB/com_banners.ini

```
@@ -10,8 +10,8 @@ COM_BANNERS_BANNER_NEW="New Banner"
 COM_BANNERS_BANNER_SAVE_SUCCESS="Banner saved."
 COM_BANNERS_BANNERS_FILTER_SEARCH_DESC="Search in banner name and alias. Prefix with ID: to search for a banner ID."
 COM_BANNERS_BANNERS_FILTER_SEARCH_LABEL="Search Banners"
-COM_BANNERS_BANNERS_HTML_PIN_BANNER="Pinned banner"
-COM_BANNERS_BANNERS_HTML_UNPIN_BANNER="Unpinned banner"
+COM_BANNERS_BANNERS_HTML_PIN_BANNER="Unpin this Banner"
+COM_BANNERS_BANNERS_HTML_UNPIN_BANNER="Pin this banner"
```

```suggestion
COM_BANNERS_BANNERS_HTML_PIN_BANNER="Pin Banner"
COM_BANNERS_BANNERS_HTML_UNPIN_BANNER="Unpin Banner"
```
Congratulations on your first contribution! The value should match the key. The fix should be at the code level to display the respective strings.